src/create_helpers.ts

```diff
@@ -507,9 +507,15 @@ export function createModule(doclet: INamespaceDoclet, nested: boolean, children
         flags |= ts.NodeFlags.NestedNamespace;
 
     if (children)
-        body = ts.createModuleBlock(children as ts.Statement[]);
-
-    const name = ts.createStringLiteral(doclet.name);
+				body = ts.createModuleBlock(children as ts.Statement[]);
```

Comment:
Looks like there is a mix of tabs and spaces here, can you normalize that?
```suggestion
        body = ts.createModuleBlock(children as ts.Statement[]);
```